src/git.rs

```diff
@@ -118,6 +118,13 @@ pub(crate) fn filter_merge_commits(commit: &git2::Commit, merges: bool) -> bool
     merges || commit.parent_count() <= 1
 }
 
+pub(crate) fn filter_revert_commits(commit: &git2::Commit, ignore_reverts: bool) -> bool {
+    if ignore_reverts {
+        return commit.message().map(|m| !m.starts_with("Revert \"")).unwrap_or(true)
+    }
+    return true
```

Comment:
```suggestion
    true
```

clippy warns here that return can be removed.

It is ok for me, I will do a clippy check and update the dependencies before creating a new release.

Reply:
yes, feel free to do it :)

Reply:
Was still failing on the format check. But it is fine I'll do a cleanup later.

Reply:
Oh I was just doing that, thank you very much 😄